ml/README.md

```diff
@@ -0,0 +1,264 @@
+
+Information about each anomaly event can then be found at the `/api/v1/anomaly_event_info` endpoint (making sure to pass the `after` and `before` params): 
+
+https://london.my-netdata.io/api/v1/anomaly_event_info?after=1638367788&before=1638367851
+
+```
+[
+    [
+        0.66,
+        "netdata.response_time|max"
+    ],
+    [
+        0.63,
+        "netdata.response_time|average"
+    ],
+    [
+        0.54,
+        "netdata.requests|requests"
+    ],
+    ...
+```
+
+**Note**: If you click the above url you will get a `null` since no such anomaly event exists as the response is just an illustrative example taken from a node that did have such an anomaly event.
+
+The query returns a list of dimension anomaly rates for all dimensions that were considered part of the detected anomaly event.
+
+**Note**: We plan to build additional anomaly detection and exploration features into both Netdata Agent and Netdata Cloud. The current endpoints are still under active development to power the upcoming features.
+
+## Configuration
+
+To enable anomaly detection:
+1. Find and open the Netdata configuration file `netdata.conf`.
+2. In the `[ml]` section, set `enabled = yes`.
+3. Restart netdata (typically `sudo systemctl restart netdata`).
+
+**Note**: If you would like to learn more about configuring Netdata please see [this guide](https://learn.netdata.cloud/guides/step-by-step/step-04).
+
+Below is a list of all the available configuration params and their default values.
+
+```
+[ml]
+	# enabled = no
+	# maximum num samples to train = 14400
+	# minimum num samples to train = 3600
+	# train every = 3600
+	# num samples to diff = 1
+	# num samples to smooth = 3
+	# num samples to lag = 5
+	# maximum number of k-means iterations = 1000
+	# dimension anomaly score threshold = 0.99
+	# host anomaly rate threshold = 0.01000
+	# minimum window size = 30.00000
+	# maximum window size = 600.00000
+	# idle window size = 30.00000
+	# window minimum anomaly rate = 0.25000
+	# anomaly event min dimension rate threshold = 0.05000
+	# hosts to skip from training = !*
+	# charts to skip from training = !system.* !cpu.* !mem.* !disk.* !disk_* !ip.* !ipv4.* !ipv6.* !net.* !net_* !netfilter.* !services.* !apps.* !groups.* !user.* !ebpf.* !netdata.* *
+```
+
+### Descriptions (min/max)
+
+- `enabled`: `yes` to enable, `no` to disable.
+- `maximum num samples to train`: (`3600`/`21600`) This is the maximum amount of time you would like to train each model on. For example, the default of `14400` would train on the preceding 4 hours of data assuming an `update every` of 1 second.
+- `minimum num samples to train`: (`900`/`21600`) This is the minimum amount of data required to be able to train a model. For example, the default of `3600` implies that once at least 1 hour of data is available for training a model will be trained, otherwise it will be skipped and checked again at the next training run.
```

Comment:
```suggestion
- `minimum num samples to train`: (`900`/`21600`) This is the minimum amount of data required to be able to train a model. For example, the default of `3600` implies that once at least 1 hour of data is available for training, a model is trained, otherwise it is skipped and checked again at the next training run.
```